### DISCUSSION TITLE
Service accounts (vs. actual users) would be great for CI and other automations

### DISCUSSION THREAD

**Discussion by nkuehn** [vercel/community#1175] [Idea]
It would be great to be able to have something like a service account (vs. a person-bound user account) that has nothing but its own API usage quota and the ability to create API tokens.   
For example, it could be tied to an organization / team all admins of that org could manage it. 

This would make automation with Vercel much cleaner, would make API limits better controllable and properly cater for leaving / coming team members without having to touch the CI setup every time certain persons come to or leave a team.  Also cleaner for compliance.

**Comment by jon-w1** [comment]
I'm encountering this concern when onboarding to Vercel as a new customer. I am only able to make login tokens for my own personal account, but I'd like to have a service account with which to perform builds/deploys from CI using the Vercel CLI.
